src/wallet/rpcdump.cpp

```diff
@@ -656,43 +656,34 @@ UniValue importwallet(const JSONRPCRequest& request)
             CPubKey pubkey = key.GetPubKey();
             assert(key.VerifyPubKey(pubkey));
             CKeyID keyid = pubkey.GetID();
-            if (pwallet->HaveKey(keyid)) {
-                pwallet->WalletLogPrintf("Skipping import of %s (key already present)\n", EncodeDestination(PKHash(keyid)));
-                continue;
-            }
+
             pwallet->WalletLogPrintf("Importing %s...\n", EncodeDestination(PKHash(keyid)));
```

Comment:
7a58ad5624b2ea9c2d2c661446cc6e046a222fca

nit, this wouldn't be logged if the key is present.

Reply:
Added a note in the commit message that this happens.